src/components/Request/NewRequestForm.Wizard.tsx

```diff
@@ -172,40 +185,64 @@ const Wizard = ({ isLoading = false, isError = false }) => {
       {state.page === "USAJobs" && <USAJobs />}
       {state.page === "DONE" && <Done gotoStep={gotoStep} />}
 
-      {!isLoading && (
-        <div className="requestWizardButtons">
+      <div className="requestWizardButtons">
+        <Button
+          disabled={state.page === "RoutingInfo" || addRequest.isLoading}
+          onClick={() => {
+            if (isValid) {
+              dispatch({ type: "prev_page", payload: methods });
+            } else {
+              trigger(undefined, { shouldFocus: true });
+            }
+          }}
+        >
+          Prev
+        </Button>
+        {state.page !== "DONE" && (
           <Button
-            disabled={state.page === "RoutingInfo" || isLoading}
-            onClick={() => {
+            appearance="primary"
+            style={{ marginLeft: "auto" }}
+            disabled={addRequest.isLoading}
+            onClick={async () => {
               if (isValid) {
-                dispatch({ type: "prev_page", payload: methods });
+                await createNewRequest().then(() =>
+                  dispatch({ type: "next_page", payload: methods })
+                );
               } else {
                 trigger(undefined, { shouldFocus: true });
               }
             }}
           >
-            Prev
+            Save and Continue
           </Button>
+        )}
+        {state.page === "DONE" && (
           <Button
+            appearance="primary"
             style={{ marginLeft: "auto" }}
-            disabled={state.page === "DONE" || isLoading}
-            onClick={() => {
+            disabled={addRequest.isLoading}
+            onClick={async () => {
               if (isValid) {
```

Comment:
Need to use another validation check for the new "Submit for Processing" button, as checking the form on the "Done" page -- will always be valid -- even if other pages have data that isn't valid/complete.  Need to leverage the same validate you use on the Send button for moving out of Draft